### ISSUE TITLE
error while using  uhooks

### ISSUE THREAD

**Issue by thipages** [WebReflection/uhooks#5]
Hi,
I try to use `uhooks`  via `import { useState } from 'uhooks'`
I got the following error
`uhooks.js:1 Uncaught TypeError: Cannot destructure property 'i' of 'h' as it is null.`

Thanks for all your works

**Comment by thipages** [comment]
I shoud be more precise, I use it in an `importmap` context

```
    <script type="importmap">
        {
            "imports": {
              "uhooks": "./node_modules/uhooks/esm.js"
            }
        }
    </script>
```

**Comment by WebReflection** [comment]
not the answer you'd like to hear, but *hooks* are very *yesterday*, signals is all I am interested about these days, so it's very likely that all my *hooks* based libraries would die out of an "*not interested in this pattern anymore, signals are going to be standardized*" reason.

TL;DR I am super sorry to disappoint you in 2024 out of a project born 4 years ago and not touched for the latest 2 years, but my time is unfortunately constrained so if these basic issues arise, I'd be better off removing myself (and the related project) from the equation.

**Comment by thipages** [comment]
Hi, 
it is ok.
thanks for the quick answer
